.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Node.js test and Build
```

Comment:
This doesn't seem to actually run...

Reply:
It will run once the workflow hits `main`. It's always the same issue when we bootstrap a project.
I linked a green build of the same commit in my `sjrd/*` repo: https://github.com/sjrd/vite-plugin-scalajs/actions/runs/4376164277